src/content/upgrades/merge/issuance/index.md

```diff
@@ -15,79 +15,25 @@ The **issuance** of ETH is the process of creating ETH that did not previously e
-
-### Pre-merge issuance breakdown {#pre-merge-issuance-breakdown}
-
-Total ETH supply: **~119,300,000 ETH** (as of Q2 2022)
-
-**Execution layer issuance:**
-
-- Estimating at 2.08 ETH per 13.3 seconds\*: **~4,930,000** ETH issued in a year
-- Currently inflating at **~4.13%** (4.93M per year / 119.3M total)
-- \*This includes the 2 ETH per canonical block, plus an average of 0.08 ETH over time from ommer blocks. Also uses 13.3 seconds, the baseline block time target without any influence from a [difficulty bomb](/glossary/#difficulty-bomb). ([See source](https://bitinfocharts.com/ethereum/))
+Ethereum validators are rewarded with ETH for attesting to the state of the chain and proposing blocks. Rewards (or penalties) are calculated and distributed at each epoch (every 6.4 minutes) based on validator performance. The validator rewards are **significantly** less than the miner rewards that used to be issued on proof-of-work (2 ETH every ~13.5 seconds), as operating a validating node is not an economically intense activity and thus does not require or warrant as high a reward.
 
 **Consensus layer issuance:**
 
-- Using 13,000,000 total ETH staked, the rate of ETH issuance is ~1600 ETH/day ([See source](https://ultrasound.money/))
-- Results in **~584,000** ETH issued in a year
-- Currently inflating at **~0.49%** (584K per year / 119.3M total)
-
-<InfoBanner>
-<strong>Total annual issuance rate: ~4.62%</strong> (4.13% + 0.49%)<br/><br/>
-<strong>~89.4%</strong> of the issuance is going to miners on the execution layer (4.13 / 4.62 * 100)<br/><br/>
-<strong>~10.6%</strong> is being issued to stakers on the consensus layer (0.49 / 4.62 * 100)
-</InfoBanner>
-
-## Post-merge {#post-merge}
-
-### Execution layer issuance {#el-issuance-post-merge}
-
-Execution layer issuance after The Merge will be zero. Proof-of-work will no longer be valid under the rules of consensus. All execution layer activity will be included in "beacon blocks", which are published and attested to by proof-of-stake validators.
-
-### Consensus layer issuance {#cl-issuance-post-merge}
+Small rewards are paid validators who attest to and propose blocks. Validator rewards accrue to _validator balances_ that are managed within the consensus layer. These are separate Ethereum accounts to the accounts we're used to on Mainnet, and until the Shanghai upgrade funds from validator accounts will not be withdrawable/transferrable. This means that although new ETH is still being issued, 100% of it will be locked from the market until this upgrade occurs. When the Shanghai upgrade is rolled out, this ETH will become available.
```

Comment:
```suggestion
Small rewards are paid to validators who attest to and propose blocks. Validator rewards accrue to _validator balances_ that are managed within the consensus layer. These are separate Ethereum accounts to the accounts we're used to on Mainnet, and until the Shanghai upgrade funds from validator accounts will not be withdrawable/transferrable. This means that although new ETH is still being issued, 100% of it will be locked from the market until this upgrade occurs. When the Shanghai upgrade is rolled out, this ETH will become available.
```

Also, until withdrawals are confirmed, we should phrase this as being proposed for the Shanghai upgrade

Reply:
```suggestion
Small rewards are paid to validators who attest to and propose blocks. Validator rewards accrue to _validator balances_ that are managed within the consensus layer. These are separate Ethereum accounts to the accounts we're used to on Mainnet, and until the Shanghai upgrade funds from validator accounts will not be withdrawable/transferrable. This means that although new ETH is still being issued, 100% of it will be locked from the market until this upgrade occurs. When withdrawals are enabled, this ETH will become available.
```